data/campaigns/Descent_Into_Darkness/scenarios/05_Orc_War.cfg

```diff
@@ -16,15 +16,23 @@
+            story=_ "The cycle of the seasons churns and summer and fall roll around once more. Malin's thoughts turn back to Parthyn frequently, where a year prior, he had still been among family and friends in the rustic little town—when he had still had a place to call home. His forced departure has turned him into a forsaken wanderer, he thinks, but the still burning image of his father being cut down and his home being destroyed is imprinted at the forefront of his mind."
             {STORYTXT_BACKGROUND travel.jpg}
         [/part]
         [part]
-            story=_ "Word spreads to the south as well, of the many battles won against the orc tribes, as Darken Volk tells Malin. As spring rolls around, a number of dark adepts, eager to aid in the fight, travel northwards and join the pair. <i>“They are quite useful in battle,”</i> the necromancer says to Malin, <i>“but none of them have even a tenth of your potential power.”</i>"
+            story=_ "Companionship and peace... perhaps they are but a farce and certainly an impossibility with the threat of the orc clans still looming. Perhaps there is no point in reminiscing. He has never been one for regrets."
             {STORYTXT_BACKGROUND travel.jpg}
         [/part]
         [part]
-            story=_ "The damage inflicted by Malin and Darken Volk over the past year forces the usually fractious local chieftains to unite in opposition. Eventually, at a narrow place in the river men call the Longlier and Elves call the Arkan-thoria, they manage to surround the humans and force a fight."
+            story=_ "Malin turns his attention to the task at hand. Darken Volk leads him through the heart of the Whitefang territory, where they terrorize and murder entire camps of orcs with their undead minions. Word spreads among the orcish towns, which begin to post guards and leave torches burning all night. These measures do little to help them."
+            {STORYTXT_BACKGROUND travel.jpg}
+        [/part]
+        [part]
+            story=_ "Word spreads to the south as well when the ever present orc raids are suddenly absent this year. The people at the frontier towns go about their days, gossiping about rumors of the undead, some with disgust, some with intrigue. When winter passes and spring comes again, a number of dark adepts travel northward and join the pair of necromancers. <i>“They can be quite useful,”</i> Darken Volk says to Malin, <i>“but none of them have even a tenth of your potential power.”</i>"
+            {STORYTXT_BACKGROUND travel.jpg}
+        [/part]
+        [part]
+            story=_ "Malin and Darken Volk continue their raids on the orcs and eventually force the usually fractious Whitefang chieftains to unite in opposition. The orcs chance the two necromancers east toward the mouth of the river men call the Longlier and Elves call the Arkan-thoria, where they manage to surround the humans and force a fight."
```

Comment:
Should "chance" be "chance upon" or "chase"? All of these could fit, but "chance" without "upon" feels odd to me.

Reply:
Chase, it's a typo. :(